client/src/Components/ContactForm.jsx

```diff
@@ -0,0 +1,94 @@
+import React from 'react';
```

Comment:
i've already seen this file in previous PR, please check branch and PR again to make sure it contains actual changes only

Reply:
I am not quite sure for this behavior, tho from what I have seen in the PR, there are changes adding redux in this branch.